Makefile

```diff
@@ -25,5 +25,6 @@ test: vendor
 	go test -timeout 1m -coverprofile=.coverage.out ./... \
 		|grep -v -E '^\?'
 
+.PHONY: vendor
 vendor:
-	dep ensure
+	go mod download
```

Comment:
Yeah I went down this route because the `make vendor` existed in the previous iteration. But you're right, I think we can just omit this altogether now. Let me update.

Reply:
Updated to remove vendor dir.